client/my-sites/domains/domain-management/list/site-domains.jsx

```diff
@@ -185,6 +186,35 @@ export class SiteDomains extends Component {
 		);
 	}
 
+	renderBreadcrumbs() {
+		const { translate } = this.props;
+
+		const item = {
+			label: 'Domains',
```

Comment:
There's a translation missing:

```suggestion
			label: translate( 'Domains' ),
```